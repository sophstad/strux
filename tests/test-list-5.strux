int main() {
	LinkedList::num l = new LinkedList::num();
	l.add(9.0);
	l.add(8.0);
	l.add(7.0);
	l.add(6.0);
	l.show();
	return 0;
}
