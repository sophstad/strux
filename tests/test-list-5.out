[ 8.000000 ] -> [ 6.000000 ]